### DISCUSSION TITLE
Redraw some existing 'completed' sprites?

### DISCUSSION THREAD

**Discussion by HippasusTwo** [Tuxemon/Tuxemon#2428] [Ideas]
I noticed @Sanglorian commissioned new art for Foofle and that it isn't the only creature with improved concept art that doesn't actually quite match the sprite. (Actually, Foofle has two different improved pieces of art.) I could also from-scratch do an upright Floofle like the original sprite if that is preferred over quadruped. Or I could try to improve on the original's shading and texture, as something about that feels off, idk.

I already tried to improve upon Dollfin awhile ago, you can see it under its 'alternative sprites' grouping. It is an actual Dollfin instead of a whale in those, though I also did a whale one in the same style. The only thing missing from that improved set is the face sprites I think, as I did an improved back and front sprite with a more prominent nose.

The art quality of many of the sprites is actually pretty decent, which makes it all the more jarring when looking at Floofle and Dollfin's current sprites which just... Lack a lot of polish.

And Shammer, I _literately_ couldn't tell it was supposed to be a dog. I thought it was a bed-frame crossed with a cow. Looking at the waybackmachine version of the old forum posts, I am pretty sure that it was intended to look a lot different originally.

Those are the ones that stand out to me the most, but I wouldn't mind doing Axylightyl as well so it better matches the style of its evo (but it isn't hideous or completely incomprehensible so it's not urgent).

Here's an example Shammer re-do, let me know if you like it or would prefer something different. I did two versions, one of them based more on what I thought the original sprite was trying to do (some kind of dark tail and leg adornments on a brown animal?) and another less complicated one that is just 'cave dog'.
![shammer-darktailed-front](https://github.com/user-attachments/assets/e42d6167-5408-4dc8-b8b3-4c130e9cfbea)
![shammer-front](https://github.com/user-attachments/assets/6c3a35ae-8dd8-499f-bb49-727b2ed41845)

**Comment by JaskRendix** [comment]
Thanks for sharing the new Shammer designs. I really like the second version. It's simple yet effective, and it captures the essence of the original sprite in a really cool way.

**Comment by Sanglorian** [comment]
Thanks @HippasusTwo - I like that redesign of Shammer (I much prefer the tail of the first one, but otherwise prefer the second one). 

The challenge of being a collaborative, community-made game is that we don't want to offend or disappoint contributors by replacing their work. But I know what you mean about there being inconsistent art styles and variable quality. 

I'm interested in what other people think about how to navigate this, but how about: 

* Anyone can propose redesigned sprites here
* If the original creator feels that the redesign does not match their original vision, the sprites will not be replaced
* Otherwise we can replace the sprites.

**Reply by HippasusTwo** [reply]
I could swap the tails pretty easily.

That's true, I don't want to offend. It also doesn't seem like a lot of the original creators are even around, though? How long a wait period do we want to give them to respond?

Actually I was thinking that having a working mod system would help a lot. Then we could let players choose to load sprites from, say, folder1_originals or folder2_remakes, and if folder2 doesn't have a corresponding remake it just loads the original.

![shammer-front3](https://github.com/user-attachments/assets/2c3da8d8-02ec-4e25-b86d-0c08963844db)
I think I like this one the best.

**Reply by Sanglorian** [reply]
> That's true, I don't want to offend. It also doesn't seem like a lot of the original creators are even around, though? How long a wait period do we want to give them to respond?

That's certainly true. I would say a week is plenty. We can always change it back later if there are concerns. 

> 
> ![shammer-front3](https://private-user-images.githubusercontent.com/155398078/367548466-2c3da8d8-02ec-4e25-b86d-0c08963844db.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjYzNTc0NzUsIm5iZiI6MTcyNjM1NzE3NSwicGF0aCI6Ii8xNTUzOTgwNzgvMzY3NTQ4NDY2LTJjM2RhOGQ4LTAyZWMtNGUyNS1iODZkLTBjMDg5NjM4NDRkYi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwOTE0JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDkxNFQyMzM5MzVaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT04ZjM1NmQ3ZDIyZGZmZTk4ZjUzNWM4ZmU5YzRiMjZjN2YwOGNiZGE5ODM5ZTYyMzRjNzAxNTI4ZjMwMWNkZjc4JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.jGuh3nP5JKB0isSu8eLQbyiFBR1uBxL9eehVGLQG88Y) I think I like this one the best.

Me too!

**Reply by HippasusTwo** [reply]
Alright!

 I don't anticipate objections with Shammer since I think the creator intended them to look different from the start anyway, so I'll continue working on it.
![shammer-menu01](https://github.com/user-attachments/assets/ddd69523-f8e9-425a-8d6e-02b8e2589ad1)
![shammer-menu02](https://github.com/user-attachments/assets/df35ca53-d479-4b02-958b-cb241ef032cb)

edit:
Getting close to a week.
![shammer-back](https://github.com/user-attachments/assets/4b1494eb-1989-4268-ae95-aa280cf21c30)

**Comment by HippasusTwo** [comment]
Dollfin

![dollfin-menu01](https://github.com/user-attachments/assets/d32f3a78-1220-49ad-8d25-aea93038fd8a)
![dollfin-menu02](https://github.com/user-attachments/assets/870193a1-f328-4cb6-8354-423bc7877aed)
![dollfin-front](https://github.com/user-attachments/assets/f4ccd2da-303d-4683-afb9-3848f1e4e81c)
![dollfin-back](https://github.com/user-attachments/assets/8a1dee8b-7a38-4d17-b35d-89537b9a65f2)

**Comment by HippasusTwo** [comment]
Alright, I did some switching (Dollfin and Shammer, I wasn't quite sure what I was doing at first but I figured it out). It is nice to have the dolphin-named character be an actual dolphin, that was rather strange and disconcerting. Although the old sprite was endearing in its own way, it is easy enough to switch back if one really wants to.

I ended up deciding to leave Shammer's skull main menu pic as-is because it looks rather cool and I hadn't noticed it before.
These were the two mons that bugged me the most. I apologize for some of my phrasing in the first post though - it occurs to me I could have stated things more gently.

I really like some of the redesigns of Moloch done by other people, and there is one that matches the concept art you first see on the page. Could we switch?

**Reply by Sanglorian** [reply]
Thanks for those, I think they look good. I've just created a new entry for the original sprite so it's preserved.

I will veto the alternative Moloch sprites though, I'm afraid! They don't capture advanced age and weight, which I want from Moloch. Leo's concept art has really set my expectations for that mon. Happy to see revisions or new sprites, of course, if anyone is inspired - but I'd like them to feel like a powerful but rounded grandfather.

**Reply by HippasusTwo** [reply]
Ah. That makes sense - I didn't realize the look was on purpose, the concept art confused me there.
